{"id":"apierror","metadata":{"name":"ApiError","description":"<h1 id=\"apierror\">ApiError</h1>\n\n<p>Raised when an API call is not successful.</p>","source":"lib/gcloud/pubsub/errors.rb#L41","resources":[],"examples":[]},"methods":[{"metadata":{"name":"code","description":"<p>The code of the error.</p>","source":"lib/gcloud/pubsub/errors.rb#L44","resources":[],"examples":[]},"params":[],"exceptions":[],"returns":[]},{"metadata":{"name":"errors","description":"<p>The errors encountered.</p>","source":"lib/gcloud/pubsub/errors.rb#L52","resources":[],"examples":[]},"params":[],"exceptions":[],"returns":[]},{"metadata":{"name":"initialize","description":"","source":"lib/gcloud/pubsub/errors.rb#L58","resources":[],"examples":[]},"params":[],"exceptions":[],"returns":[{"types":["ApiError"],"description":"a new instance of ApiError"}]},{"metadata":{"name":"klass_for","description":"","source":"lib/gcloud/pubsub/errors.rb#L71","resources":[],"examples":[]},"params":[],"exceptions":[],"returns":[]}]}